changelogs/fragments/5632-vdo-Use-yaml-safe-load-instead-of-yaml-load.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - The vdo module now uses yaml.safe_load() to parse command output instead of the deprecated yaml.load() which is potentially unsafe. Using yaml.load() without explicitely setting a Loader= is also an error in pyYAML 6.0.
```

Comment:
```suggestion
  - vdo - now uses ``yaml.safe_load()`` to parse command output instead of the deprecated ``yaml.load()`` which is potentially unsafe. Using ``yaml.load()`` without explicitely setting a ``Loader=`` is also an error in pyYAML 6.0 (https://github.com/ansible-collections/community.general/pull/5632).
```